frontends/precinct-scanner/src/screens/scan_jam_screen.tsx

```diff
@@ -11,7 +11,7 @@ export function ScanJamScreen(): JSX.Element {
     <ScreenMainCenterChild infoBar={false}>
       <TimesCircle />
       <CenteredLargeProse>
-        <h1>Scanner Jammed</h1>
```

Comment:
This the most important screen to convey that a ballot was not counted explicitly.